easyconfigs/a/ALAMODE/ALAMODE-1.4.2-foss-2022b.eb

```diff
@@ -0,0 +1,56 @@
+# with thanks to ComputeCanada for the initial easyconfig for 1.0.2-iomkl-2017.5.211.eb
+# updated for 1.4.2 by BEAR Software team at University of Birmingham
+easyblock = "CMakeMakeCp"
+
+name = 'ALAMODE'
+version = '1.4.2'
+
+homepage = 'http://alamode.readthedocs.io/en/latest/index.html'
+description = """ALAMODE is an open source software designed for analyzing lattice anharmonicity
+ and lattice thermal conductivity of solids. By using an external DFT package such as VASP and
+ Quantum ESPRESSO, you can extract harmonic and anharmonic force constants straightforwardly with ALAMODE.
+ Using the calculated anharmonic force constants, you can also estimate lattice thermal conductivity,
+ phonon linewidth, and other anharmonic phonon properties from first principles.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://github.com/ttadano/alamode/archive/']
+sources = ['v.%(version)s.tar.gz']
+checksums = ['18fb80a183c2a9b68488aeeccd6efa95e8c21d4cb21b08319a557389d55c067a']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('Boost', '1.81.0'),
+    ('Eigen', '3.4.0'),
+    ('spglib', '2.0.2'),
+    ('FFTW', '3.3.10'),
+    ('HDF5', '1.14.0'),
+    ('Python', '3.10.8'),
+]
+
+configopts = ' '.join([
+    "-DCMAKE_BUILD_TYPE=Release",
+    "-D_HDF5=yes",
```

Comment:
Is this correct? It looks odd.

Reply:
Yes it should - I erroneously used the make option here.
fixed in a031b01